plugins/typo_squatter/unit_test/test_entropy.py

```diff
@@ -0,0 +1,12 @@
+from unittest import TestCase
+from komand_typo_squatter.util.utils import entropy
+
+
+class TestEntropy(TestCase):
+    def test_entropy(self):
+        result = entropy("test.example.com")
+        self.assertEqual(3.327819531114783, result)
+
+    def test_entropy2(self):
+        result = entropy("login.test.example_domain.com")
+        self.assertEqual(3.8573068568981284, result)
```

Comment:
are there significance to these numbers or is this just what happens to pop out of the entropy function? Is the point here to show the longer domain with more characters had a higher entropy score?

Reply:
These are the numbers returned by the entropy method. Yes, with longer domains the entropy increases.